tests/test_resultinfo.py

```diff
@@ -148,7 +161,28 @@ def test_print_available_result_with_qualifiers(cfx_heating_coil, server_type):
 )
 def test_print_result_info_with_qualifiers(cfx_heating_coil, server_type):
     model = Model(cfx_heating_coil(server=server_type), server=server_type)
-    ref = """Static analysis
+    if SERVERS_VERSION_GREATER_THAN_OR_EQUAL_TO_10_0:
+        ref = """Static analysis
+Unit system: Custom: m, kg, N, s, V, A, K
```

Comment:
@TheoMeunierAnsys I' sorry I do not see the difference with the initial reference